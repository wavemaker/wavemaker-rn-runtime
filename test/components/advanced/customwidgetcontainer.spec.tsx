import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmCustomWidgetContainer from '@wavemaker/app-rn-runtime/components/advanced/CustomWidgetContainer/CustomWidgetContainer.component';

describe('Test CustomWidgetContainer component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmCustomWidgetContainer name="test_CustomWidgetContainer"></WmCustomWidgetContainer>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});