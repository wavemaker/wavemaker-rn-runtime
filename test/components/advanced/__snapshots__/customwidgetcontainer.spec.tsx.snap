// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CustomWidgetContainer component Check validity of sample component 1`] = `
<View
  style={{}}
/>
`;
